src/TagNameMatcher.php

```diff
@@ -39,9 +39,13 @@ public function describeTo( Description $description ) {
 	 * @return bool
 	 */
 	protected function matchesSafelyWithDiagnosticDescription( $item, Description $mismatchDescription ) {
+		if ( $this->tagNameMatcher->matches( $item->tagName ) ) {
+			return true;
```

Comment:
This seems to change behavior (but probably for good)… was this on purpose?

Reply:
What behavior do you mean? The idea is: If the match succeeds, it is pointless to "describe" a mismatch that never happened.

Reply:
Thought so… just wanted to make sure you changed that on purpose :)